taxi/tests/tests_models.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+# Create your tests here.
+class ModelsTestCase(TestCase):
+    def test_manufacturer_str_and_dani(self):
```

Comment:
There is a typo in the method name `test_manufacturer_str_and_dani`. It should be `test_manufacturer_str` to correctly reflect the purpose of the test.